lighty-core/dependency-versions/pom.xml

```diff
@@ -98,8 +98,8 @@
             </dependency>
```

Comment:
Please add here also versioning for slf4j-api. Current version 1.7.32 is pulled from karaf-bom dependency.
```xml
<dependency>
    <groupId>org.slf4j</groupId>
    <artifactId>slf4j-api</artifactId>
    <version>1.7.35</version>
</dependency>
````
It would be better to have it uniform